e2e-tests/src/test/java/com/swissquote/github/carnotzet/e2e/test/ExtensionExamplesTest.java

```diff
@@ -0,0 +1,51 @@
+package com.swissquote.github.carnotzet.e2e.test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.hamcrest.CoreMatchers;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.github.swissquote.carnotzet.core.runtime.CommandRunner;
+
+public class ExtensionExamplesTest {
+
+	private static File logFile;
+	private static Verifier mavenVerifier;
+
+	@BeforeClass
+	public static void setUp() throws IOException, VerificationException {
+		Path projectPom = FileSystems.getDefault().getPath("../examples/voting-worker").toRealPath();
```

Comment:
Please run redis example module instead to reduce test execution time. (you'll have to move the maven plugin config)